packages/ui-portal/src/Portal/ReactPortal.tsx

```diff
@@ -21,21 +21,28 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import React from 'react'
+import React, { HTMLAttributes } from 'react'
 import PropTypes from 'prop-types'
 import ReactDOM from 'react-dom'
 
 import { passthroughProps } from '@instructure/ui-react-utils'
 import { bidirectional, BidirectionalProps } from '@instructure/ui-i18n'
 import { element } from '@instructure/ui-prop-types'
 
+import { PortalNode } from './PortalType'
+
 type Props = {
   open?: boolean
-  onOpen?: (DOMNode: HTMLSpanElement | null) => any
-  onClose?: (...args: any[]) => any
-  mountNode?: any // TODO: PropTypes.oneOfType([element, PropTypes.func]),
+  onOpen?: (DOMNode: PortalNode) => any
+  onClose?: () => any
```

Comment:
I think it would be nicer if we'd have `void` as a return type for these callbacks since we're not awaiting any return value from these methods.

Reply:
done, thanks!